data_steward/cdr_cleaner/manual_cleaning_rules/survey_version_info.py

```diff
@@ -226,22 +227,24 @@ def add_console_logging(add_handler):
         required=True,
         help='Name of the table cotaining the cope survey mapping information')
     parser.add_argument(
-        '--mapping_dataset',
-        dest='mapping_dataset_id',
+        '-q',
+        '--deid_questionnaire_response_map_dataset',
         action='store',
-        help=('Dataset name for the dataset containing deid mapping tables.  '
-              'For example, _deid_map and _deid_questionnaire_response_id.'))
+        dest='deid_questionnaire_response_map_dataset',
+        help=
+        'Identifies the dataset containing the _deid_questionnaire_response_map lookup table',
+        required=True)
 
     ARGS = parser.parse_args()
     if not ARGS.mapping_dataset_id:
```

Comment:
This conditional will never pass since there is no longer a `mapping_dataset_id` argument. It should either be removed or changed to reference the new argument `deid_questionnaire_response_map_dataset` (although that argument is already required, so this may not be necessary).